exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,21 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0;
+}
+
+function isBetween5and10(num) {
+  return num > 5 && num <= 10;
+}
```

Comment:
This should return true for the number 5 - it would return false. You need to change '>'